integration_tests/pages/induction/CheckYourAnswersPage.ts

```diff
@@ -242,9 +241,8 @@ export default class CheckYourAnswersPage extends Page {
     return Page.verifyOnPage(AffectAbilityToWorkPage)
   }
 
-  submitPage(): WorkAndInterestsPage {
+  submitPage() {
```

Comment:
I had to change (remove) the return type of the submitPage function on the Check Your Answers cypress page class because different use cases of it result in different pages:

* The Update journey will result in the user being sent back to the PLP Work And Interests page
* The Create journey will result in the user being sent to:
  * The PLP Create Goals page if the prisoner does not have any Goals yet
  * The PLP Overview page if the prisoner already has Goals